plugins/woocommerce/tests/e2e-pw/test-data/data.js

```diff
@@ -22,6 +22,8 @@ const customer = {
 	username: CUSTOMER_USER ?? 'customer',
 	password: CUSTOMER_PASSWORD ?? 'password',
 	email: CUSTOMER_USER_EMAIL ?? 'customer@woocommercecoree2etestsuite.com',
+	first_name: 'Jane',
+	last_name: 'Smith',
```

Comment:
I found one `shopper` test that uses hard-coded values for the customer's first and last name. I centralized access to these values here.